src/sentry/tasks/sentry_apps.py

```diff
@@ -115,6 +115,15 @@ def send_alert_event(event, rule, sentry_app_id):
         data=request_data.body,
         headers=request_data.headers,
         timeout=5,
+        metrics={
+            'key': 'webhook',
+            'instance': 'sentry.tasks.sentry_apps.send_alert_event',
+            'tags': {
+                'integration_platform': True,
+                'event': 'event_alert.triggered',
+                'sentry_app': sentry_app.slug,
```

Comment:
what's the cardinality of sentry_apps ? (every new tag is an additional metric on datadog afaik). I think you should just quickly confirm with james or matt whether we are fine in having a new metric per app.